FitNesseRoot/DbFit/AcceptanceTests/JavaTests/DerbyTests/FlowMode/SimpleStoredProc.wiki

```diff
@@ -8,3 +8,15 @@ This stored proc tests output parameters - !-CalcLength-! returns string length
 |mika                    |4             |
 |paradajz                |8             |
 |saeiluhrweurhfi7fhi7rhgf|24            |
+
+|Execute Procedure       |!-CalcLength-!|
+|name                    |str length?   |
+|mika                    |>>mklen       |
+|paradajz                |>>>prlen      |
+
+|Execute Procedure       |!-CalcLength-!|
+|name                    |str length?   |
+|mika                    |<<mklen       |
+|paradajz                |<<<prlen      |
+|<<mklen                 |1             |
+|<<<prlen                |1             |
```

Comment:
Can we include some exec proc/func tests for other DBMSs right now seeing as we now have 5 in the CI?